### PR TITLE
Don't compile story files and remove emotion sourcemaps

### PR DESCRIPTION
**Pull request by tmeasday**
<details>
  <summary>📦 Published PR as canary version: <code>7.3.7-canary.358.b16e1ca.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @storybook/design-system@7.3.7-canary.358.b16e1ca.0
  # or 
  yarn add @storybook/design-system@7.3.7-canary.358.b16e1ca.0
  ```
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "patch-package": "patch-package",
-    "build": "yarn build-images && yarn patch-package && babel src -d dist --extensions \".js,.jsx,.ts,.tsx\" --ignore \"**/*.test.js\" --ignore \"**/*.stories.js\"",
+    "build": "yarn build-images && yarn patch-package && babel src -d dist --extensions \".js,.jsx,.ts,.tsx\" --ignore \"**/*.stories.tsx\" --ignore \"**/*.stories.js\"",
```
Comment (shilman) [inline package.json:20]: `*.stories.*`?

**Review by shilman — APPROVED** [review]
LGTM with maybe one small change

File: package.json (hunk 1 above)
Comment (tmeasday) [reply package.json:20]: Doesn't work. No idea why.

Comment (shilman) [reply package.json:20]: 🤷‍♂️

**Pull request merged**